src/test/java/tests/BaseTestClass.java

```diff
@@ -241,6 +242,18 @@ SubmitResponseBean submitForm(String requestPath, String sessionId) throws Excep
                 SubmitResponseBean.class);
     }
 
+    SubmitResponseBean submitFormAnswers(Map<String, Object> answers, String sessionId) throws Exception {
+        SubmitRequestBean submitRequestBean = mapper.readValue
+                (FileUtils.getFile(this.getClass(), "requests/submit/submit_request.json"), SubmitRequestBean.class);
```

Comment:
might want to make this test specific or have the file name be an arg, otherwise not sure how this'll be reused that easily by someone without context